modules/react/breadcrumbs/stories/Breadcrumbs.stories.mdx

```diff
@@ -0,0 +1,259 @@
+
+# Canvas Kit Breadcrumbs
+
+Breadcrumbs is a
+[compound component](/getting-started/for-developers/resources/compound-components/) that allows
+users to keep track and maintain awareness of their location as they navigate through pages,
+folders, files, etc.
+
+[> Workday Design Reference](https://design.workday.com/components/navigation/breadcrumbs)
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-react
+```
+
+## Usage
+
+### Basic Example
+
+`Breadcrumbs` includes a container `Breadcrumbs` component and the following subcomponents which can
+be composed in a variety of ways: `Breadcrumbs.List`, `Breadcrumbs.Item`, `Breadcrumbs.CurrentItem`
+and `Breadcrumbs.Link`.
+
+Accessibility is built into Breadcrumbs components in a way that allows you to create an inclusive
+experience with little additional configuration. The only ARIA label you'll need to add is the
+`aria-label` for `Breadcrumbs.List` as seen in the example. And all links require `href`s to be
+properly identified. All other accessible attributes are baked into the components.
+
+<ExampleCodeBlock code={Basic} />
+
+### Collapsed List
+
+`Breadcrumbs` is a responsive component based on the width of its container. If the rendered links
+exceed the width of the `Breadcrumbs.List`, an overflow menu will be rendered. This only works
+against the dynamic API where you give the `BreadcrumbsModel` an array of items to be rendered. The
+dynamic API handles the React `key` for you based on the item's identifier. The dynamic API requires
+either an `id` on each item object or a `getId` function that returns an identifier based on the
+item. The below example uses an `id` property on each item.
+
+The dynamic API takes in any object, but since nothing is known about your object, a
+[render prop](https://reactjs.org/docs/render-props.html) is necessary to instruct a list how it
+should render.
+
+<ExampleCodeBlock code={Collapsed} />
+
+### Right-to-Left (RTL)
+
+`Breadcrumbs` has bidirectional support out of the box. That means outside of setting the content
+direction in your application's Canvas theme, you don't need to do anything else to make it work.
+All you need to supply is the translated text for items and ARIA labels.
+
+<ExampleCodeBlock code={RTLCollapsibleList} />
+
+## Components
+
+### Breadcrumbs
+
+#### Usage
+
+`Breadcrumbs` is a container component that is responsible for creating a `BreadcrumbsModel` and
+sharing it with its subcomponents using React context. It does not represent a real element.
+
+```tsx
+<Breadcrumbs items={[]}>{/* Child components */}</Breadcrumbs>
+```
+
+Alternatively, you may pass in a model using the hoisted model pattern.
+
+```tsx
+const model = useBreadcrumbsModel({
+  items: [],
+});
+
+<Breadcrumbs model={model}>{/* Child components */}</Breadcrumbs>;
+```
+
+#### Props
+
+Note that if you pass in a `model` configured with `useBreadcrumbsModel`, configuration props passed
+to `Breadcrumbs` will be ignored.
+
```

Comment:
Adding a missing prop table for `Breadcrumbs`, though perhaps you omitted it intentionally since it's quite long?

We're showing the full props table for [`Tabs`](https://workday.github.io/canvas-kit/?path=/docs/components-containers-tabs-react--basic#props) which also uses `Menu`. I say we show it here as well for consistency since it's not obscenely long.
```suggestion
Note that if you pass in a `model` configured with `useBreadcrumbsModel`, configuration props passed
to `Breadcrumbs` will be ignored.

<ArgsTable of={Breadcrumbs} />

```